lms/djangoapps/courseware/views/views.py

```diff
@@ -69,7 +68,6 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey, UsageKey
 from openedx.core.djangoapps.catalog.utils import get_programs, get_programs_with_type
-from openedx.core.djangoapps.certificates import api as auto_certs_api
```

Comment:
Note: this is already imported at https://github.com/edx/edx-platform/pull/16376/files#diff-2749747f3edc5443c734c6a6546e91d1R77